src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -169,10 +169,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('set_value', options_list=['--set'], help="Task value in '--set name[=value]' format. Multiples supported by passing --set multiple times.", action='append', validator=validate_set)
         c.argument('set_secret', help="Secret task value in '--set-secret name[=value]' format. Multiples supported by passing --set-secret multiple times.", action='append', validator=validate_set_secret)
 
-        # Source Trigger parameters
+        # Trigger parameters
         c.argument('source_trigger_name', help="The name of the source trigger.")
         c.argument('commit_trigger_enabled', help="Indicates whether the source control commit trigger is enabled.", arg_type=get_three_state_flag())
         c.argument('pull_request_trigger_enabled', help="Indicates whether the source control pull request trigger is enabled.", arg_type=get_three_state_flag())
+        c.argument('schedules', nargs='+', help="Space-separated list of schedules represented as cron expressions for the timer triggers. An optional trigger name can be specified using `--schedules name:schedule` format.")
```

Comment:
is this comment correct, i.e. it doesn't look like it will accept anything space-separated, rather it should be specified by chaining `--schedules` multiple times.

I would also prefer using `--schedule` when chaining instead of the plural

Reply:
Agreed singular is better.

Reply:
The `nargs='+'` allows you to pass a space separated list of values.

Reply:
So do we want the user to provide `--schedules "* */12 * * *" "30 9 * * *"` or `--schedule "* */12 * * *" --schedule  "30 9 * * *"` for create?